contribute/contribute.rst

```diff
@@ -320,6 +327,23 @@ Be sure to check that the resulting build behaves as expected over:
 In (1.) above a db wipe is initiated helping to test the self-start code capability.
 See the initrock script and it's systemd trigger service: rockstor-init.
 
+We also have **automated tests** in place that cover our API's and core critical path functionality.
+It is expected that any changes to critical path code e.g. fs management / updates / Web-UI / services;
+include counterpart contributions to prove the expected function, if required.
+This is an oft neglected element in software development;
+but we are attempting to better our own standing in this regard.
+
+The following will run all tests post the above detailed source install:
```

Comment:
Style suggestion:
```suggestion
The following will run all tests following the source installation detailed above:
```

Reply:
OK; we can pull off 2 x 'following' in the same sentence :). And reads better overall anyway. Thanks.